x-pack/legacy/plugins/integrations_manager/index.ts

```diff
@@ -83,5 +100,16 @@ const pluginOptions: LegacyPluginOptions = {
   isEnabled: false,
 };
 
+export const getConfigSchema = (Joi: typeof JoiNamespace) => {
+  const IntegrationsManagerConfigSchema = Joi.object({
```

Comment:
Why `joi` here and `@kbn/config-schema` elsewhere? Can we define this using the same utility/library?

Reply:
`joi` is here because I'm using the legacy config service to shim the NP config service, so I'm using the legacy way to define the schema.

It will disappear completely once we have a way to access the NP config service directly. I don't think it's worth the effort to abstract this away, as it's hopefully a very temporary situation